modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,7 +33,88 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
 
 You can specify the version of the proxy you want to use by clicking the *Advanced Options* drop-down menu and selecting the proxy version that best covers your needs.
 
-See xref:proxy-deploy-raml-oas-proxy.adoc[Building a RAML/OAS API Proxy] for more information.
+== API Proxy Validations
+
+You can configure API proxies to perform validations on all incoming requests against the API definition you design. Depending on the Mule runtime and API Proxy version you installed, you can modify these validation settings in API Manager and redeploy the proxy in Mule Rumtime. 
```

Comment:
Typo here. `Mule Rumtime`

Reply:
Wouldn't it be nice if it were "rumtime". :-) Good catch! Thanks!